docs/source/chaincode_lifecycle.md

```diff
@@ -134,12 +134,13 @@ consistent across organizations:
 - **Name:** The name that applications will use when invoking the chaincode.
 - **Version:** A version number or value associated with a given chaincodes
   package. If you upgrade the chaincode binaries, you need to change your
-  chaincode version as well.
-- **Sequence:** The number of times the chaincode has been defined. This value
+  chaincode version as well. Versions may contain any characters,
+  but it is typical to use a format such as `v1.2.3`.
```

Comment:
add 'that this is treated as a tag by the peer. it does not do any semantic version or other checking'

or something similar; so people know it's 'just an identifier